test/provision/k8s_install.sh

```diff
@@ -47,8 +48,14 @@ sudo rm /var/lib/apt/lists/lock || true
 retry_function "wget https://packages.cloud.google.com/apt/doc/apt-key.gpg"
 apt-key add apt-key.gpg
 
+sudo swapoff -a
```

Comment:
Just curious, what do we need this for?

Reply:
Is mandatory for kubernetes

Reply:
What makes this mandatory? Please add a comment in the script to specify why this is necessary so future editors of the script have appropriate context.